cb/genesis/cb.vp

```diff
@@ -2,10 +2,10 @@
 //; my $width  = parameter(Name=>'width', val=> 16, doc=>'Bus width for CB');
```

Comment:
let's not obfuscate this change w/ style/formatting changes to the genesis code. If we want to do that in a separate pass, lets.